lnbits/db.py

```diff
@@ -47,27 +49,55 @@ def _parse_timestamp(value, _):
             lambda value, curs: float(value) if value is not None else None,
         )
     )
-    register_type(
-        new_type(
-            (1082, 1083, 1266),
-            "DATE2INT",
-            lambda value, curs: time.mktime(value.timetuple())
-            if value is not None
-            else None,
-        )
-    )
```

Comment:
this is somewhat unrelated (postgres only), I thought I removed it, but seems like I forgot to.
I had some initial suspicions on it as described above which proved to be true when working on https://github.com/lnbits/lnbits/pull/1756 because psycopg tried to convert postgres dates with our provided function - which failed and resulted in `None` values.